ee/lib/stripe/client.ts

```diff
@@ -18,7 +18,12 @@ const getStripe = (userPublicKey?: string) => {
   return stripePromise;
 };
 
-export function createPaymentLink(paymentUid: string, name?: string, date?: string, absolute = true): string {
+export function createPaymentLink(
+  paymentUid: string,
+  name?: string | null,
+  date?: string,
+  absolute = true
+): string {
```

Comment:
Maybe a risky refactor, but this would be better as an options object

```suggestion
export function createPaymentLink(opts: {
  paymentUid: string;
  name?: string | null;
  date?: string;
  absolute?: boolean;
}): string {
```

Reply:
@KATT pretty doable, `createPaymentLink`is only being used twice. Thanks for the feedback.